### ISSUE TITLE
a11y: pages need unique titles

### ISSUE THREAD

**Issue by eatyourgreens** [OxfordRSE/gutenberg#357]
Every page has the same title: OxRSE Training

Pages should have unique titles eg. Software Architecture and Design. 

https://train.rse.ox.ac.uk/material/HPCu/software_architecture_and_design

Ideally, each page should also have a h1 heading that reflects the page title too. 

Blind users will use the page title to orient themselves within the site, and page headings to navigate within the page. 

The page title is also used to distinguish pages in browser history, and used by search engines to link back to a page.
